projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-input/search-ai-input.component.spec.ts

```diff
@@ -136,8 +157,8 @@ describe('SearchAiInputComponent', () => {
       });
 
       it('should have correct agent names', async () => {
-        expect(await options[0].getText()).toBe('HA HR Agent');
```

Comment:
@jacekpluta as we discussed on one of our calls - if there is no avatar then by default initials are displayed. So in case when we have initials then we get following text "HA HR Agent" because "HA" is initials of HR Agents. This function reads whole text from option so it includes initials as well